src/explore-education-statistics-frontend/jest.config.js

```diff
@@ -11,7 +11,7 @@ const config = {
     url: 'http://localhost/',
   },
   transform: {
-    '^.+\\.(js|jsx|ts|tsx)$': 'babel-jest',
+    '^.+\\.(t|j)sx?$': '@swc/jest',
```

Comment:
If we're using SWC for common and the frontend, we could just use it for admin as well?

Reply:
There's lots of incompatibility between swc & CRA it seems as mentioned which made it not possible to use